schedulezoomtalks.py

```diff
@@ -0,0 +1,146 @@
+    # password length to max 10 characters.
+    zoom_password = secrets.token_urlsafe(10)
+
+    # Form the talk registration body
+    request_body =
+    {
+      "topic": "Speakers\' corner talk by %s"%(talk.get("name")),
+      "type": 2, # Scheduled meeting
+      "start_time": talk["time"],
+      "timezone": "UTC",
+      "duration": 60,
+      "schedule_for": SPEAKERS_CORNER_USER_ID,
+      "password": zoom_password,
+
+      # Meeting settings
+      "settings": {
+        "host_video": True,
+        "participant_video": False,
+        "cn_meeting": False,  # Host the meeting in China?
+        "in_meeting": False,  # Host the meeting in India?
+        "join_before_host": False, # This will be switched to True shortly
+                                   # before the meeting starts by the VSF bot.
+                                   # It will also be switched back to False
+                                   # after the meetings ends.
+        "mute_upon_entry": True,
+        "watermark": False,  # Don't add a watermark when screensharing
+        "use_pmi": False, # Don't use Personal Meeting ID, but generate one
+        "approval_type": 0, # Automatically approve
+        "close_registration" : True, # Close registration after event date
+        "waiting_room" : False,    # No waiting room
+        "audio": "both",
+        "auto_recording": "cloud",
+        "enforce_login": False,
+        "alternative_hosts": "",
+        "registrants_email_notification": True,
+        "contact_email": "vsf@virtualscienceforum.org",
+      },
+
+      "questions": [
+        {"field_name": "First Name", "required": True},
+        {"field_name": "Last Name", "required": True},
+        {"field_name": "Email Address", "required": True},
+        {"field_name": "Confirm Email Address", "required": True},
+        {"field_name": "Organization", "required": True},
+      ],
+
+      "custom_questions": [
+        {
+          "title": "May we contact you about future Virtual Science Forum events?",
+          "type": "single", # short or single
+          "answers": ["Yes", "No"], # only single
+          "required": true
+        },
+        {
+          "title": "How did you hear about the Virtual Science Forum?",
+          "type": "single", # short or single
+          "answers": ["Email list",
+                      "One of the organizers",
+                      "A colleague (not an organizer)",
+                      "Other"],
+          "required": true
+        },
+        {
+          "title": "Please confirm you have read the participant instructions: \
+                    http://virtualscienceforum.org/#/attendeeguide*",
+          "type": "short", # short or single
+          "required": true
+        },
+      ]
+    }
+
+    response = zoom_request(
+        requests.post,
+        f"{ZOOM_API}users/{user_id}/meetings",
+        params={"body":request_body}
+    )
+
+    return response.id, zoom_password
```

Comment:
We don't need to return the password AFAIK. Everyone gets the full link after registration.